### PR TITLE
sunxi: add csi video support for nanopi-neo-air

### PR DESCRIPTION
**Pull request by mpromonet**
Hi,

This pull request is adding support of camera for nano pi air.

Best Regards,
Michel.

### CODE REVIEW

**Comment by hauke** [comment]
Please send this also to the upstream Linux kernel to integrate it upstream.

File: target/linux/sunxi/patches-6.6/451-sunxi-h3-add-camera-support-for-nanopi-neo-air.patch
Comment (hauke) [inline target/linux/sunxi/patches-6.6/451-sunxi-h3-add-camera-support-for-nanopi-neo-air.patch:1]: Please add a git header.

You can do this by cloning the Linux kernel, adding this patch with `patch -p1 < filename` then commit it with a commit message and then export it again with `git format-patch`.

**Comment by hauke** [comment]
The changes are looking fine. They are similar to other camera drivers in dts files.

File: target/linux/sunxi/patches-6.6/451-sunxi-h3-add-camera-support-for-nanopi-neo-air.patch
Comment (mpromonet) [reply target/linux/sunxi/patches-6.6/451-sunxi-h3-add-camera-support-for-nanopi-neo-air.patch:1]: Hi @hauke
Thanks for the review,
patch updated
Best Regards,
Michel.

**Comment by mpromonet** [comment]
> Please send this also to the upstream Linux kernel to integrate it upstream.

I submitted in https://github.com/torvalds/linux/pull/918

**Comment by hauke** [comment]
Please refresh the patch using: `make target/{clean,refresh} V=99`.

**Pull request merged**